salt/docs/config/nginx.docs-backend.conf

```diff
@@ -36,12 +36,111 @@ server {
     }
 
     # Smooth the switch between versions by mapping old files to their new location
-    location ~ ^/([a-z-]*/)?(3|3.5|3.6|3.7|3.8)/library/sets.html$ {
-        return 301 https://$host/$1$2/library/stdtypes.html#set-types-set-frozenset;
-    }
     location ~ ^/([a-z-]*/)?(3|3.6|3.7|3.8)/library/email.util.html$ {
         return 301 https://$host/$1$2/library/email.utils.html;
     }
+    location ~ ^/([a-z-]*/)?(3|3.5|3.6|3.7|3.8|3.9|3.10)/c-api/(class|cobject|int|string).html$ {
+        return 301 https://$host/$1$2/;
```

Comment:
```suggestion
        return 301 https://$host/$1$2/c-api/;
```
maybe?

Reply:
I think that makes sense. What do you think about linking them to their replacements instead?

https://docs.python.org/2/c-api/cobject.html -> https://docs.python.org/3.9/c-api/capsule.html

https://docs.python.org/2/c-api/class.html -> https://docs.python.org/3/c-api/type.html

https://docs.python.org/2/c-api/int.html -> https://docs.python.org/3/c-api/long.html

https://docs.python.org/2/c-api/string.html -> https://docs.python.org/3/c-api/bytes.html